src/Scene.js

```diff
@@ -96,7 +96,10 @@ class Scene {
         val = val || undefined;
         if (val) {
           const el = _.isString(val) ? Util.elements(val)[0] : val;
-          if (el && el.parentNode) {
+          if ( el === undefined ) {
+            console.error( `ERROR triggerElement: '${val}' is still undefined! Error not being thrown.`)
+          }
+          if (el !== undefined && el.parentNode) {

```

Comment:
Can we reduce to just this line?

Reply:
Of course. This is my first pull request. I was just using it to show you. I love this plug-in BTW.

Reply:
Thanks! I just ported it to es6 obvs 😄 

If your ok to update the PR I'll merge!